### ISSUE TITLE
Package maestro-cli-mac-intel.tar.gz and maestro-cli-mac-intel.pkg not a Intel CPU arch

### ISSUE THREAD

**Issue by cherts** [PrajwalVandana/maestro-cli#13]
Hello,

Package maestro-cli-mac-intel.tar.gz and maestro-cli-mac-intel.pkg not a Intel CPU arch

How to reproduce the problem:
```
$ sysctl -n machdep.cpu.brand_string
Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz

$ wget https://github.com/PrajwalVandana/maestro-cli/releases/download/2.0.2/maestro-cli-mac-intel.tar.gz
$ tar -vzxf maestro-cli-mac-intel.tar.gz 
$ cd dist/maestro/
$ ./maestro 
bash: ./maestro: Bad CPU type in executable

$ file ./maestro
./maestro: Mach-O 64-bit executable arm64
```

Ooops, something went wrong :(

**Comment by PrajwalVandana** [comment]
Might be an issue in my GitHub action, I’ll check it out!

**Comment by PrajwalVandana** [comment]
Found the issue! I switched the names for Intel and Apple Silicon 💀😭 The `maestro-cli-apple-silicon` files are the Intel ones

Will fix soon, but for now just try that

**Comment by PrajwalVandana** [comment]
Fixed in release 2.0.2 manually (the names of the links in the release are correct, but the downloaded file will still have the opposite name; this will be fixed in future releases) and in GitHub Action workflow for future releases
